Triangulation_3/include/CGAL/Triangulation_3_to_lcc.h

```diff
@@ -52,7 +70,7 @@ namespace CGAL {
     for (TVertex_iterator itv = atr.vertices_begin();
          itv != atr.vertices_end(); ++itv)
     {
-      TV[itv] = alcc.create_vertex_attribute(itv->point());
+      TV[itv] = alcc.create_vertex_attribute(internal::Get_point<typename Triangulation::Point>::run(itv->point()));
```

Comment:
I only notice now, but this seems like it should just have been `atr->construct_point(atr->point(itv))` (might be worth it to add an overload to have `Tr::construct_point(Vertex_handle)` directly) without requiring that full class.

Reply:
I don't find the construct_point method in any triangulation 3 concept, but only in Periodic triangulation. Is this method exists for all triangulation 3 in cgal ?

Reply:
https://github.com/CGAL/cgal/blob/master/Triangulation_3/include/CGAL/Triangulation_3.h#L568 is the particular function. It is just a call to the model of `ConstructPoint_3` provided in the traits, and this functor is required by the concept `TriangulationTraits_3`.

Reply:
Ok, thanks @MaelRL ; will try...